AndroidManifest.xml

```diff
@@ -12,6 +12,8 @@
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
 
+    <uses-permission android:name="com.android.launcher.permission.INSTALL_SHORTCUT" />
```

Comment:
Awesome you are trying that! :)

Is this a "dangerous" permission that we need to ask the user to grant us?

Do you want to remove the widget or offer it as an alternative option?